static/app/components/stream/groupChart.tsx

```diff
@@ -54,6 +60,29 @@ function GroupChart({
       seriesName: t('Events'),
       data: stats.map(point => ({name: point[0] * 1000, value: point[1]})),
```

Comment:
would prefer to conditionally add the markline to this series instead of adding another